src/sybil/table_block_io.go

```diff
@@ -196,7 +196,7 @@ func (t *Table) LoadBlockInfo(dirname string) *SavedColumnInfo {
 
 // TODO: have this only pull the blocks into column format and not materialize
 // the columns immediately
-func (t *Table) LoadBlockFromDir(dirname string, loadSpec *LoadSpec, loadRecords bool) *TableBlock {
+func (t *Table) LoadBlockFromDir(dirname string, loadSpec *LoadSpec, loadRecords bool, replacements map[string]StrReplace) *TableBlock {
```

Comment:
the whole diff is good, just not a fan of adding this 4th param on. i'm even wondering if we should have another API, like: 
`LoadBlockFromDirWithSpec()` that takes the loadspec and a function that doesn't  (so people can be exposed to the lower param count API when they need it) since there's only one callsite (in this diff) that passes in a spec

Reply:
yeah this also makes me a bit uncomfortable, happy to factor that out.

Reply:
I think there's a slightly larger cleanup/refactor here so I'll move forward with this and we can clean this up in the future.